apps/qtmozembed/declarativewebpage.cpp

```diff
@@ -63,7 +63,6 @@ DeclarativeWebPage::DeclarativeWebPage(QObject *parent)
     , m_fullScreenHeight(0.f)
     , m_toolbarHeight(0.f)
     , m_virtualKeyboardMargin(0.f)
-    , m_marginChangeThrottleTimer(0)
```

Comment:
Throttling used to play a role and might be still needed but hopefully we can drop it. This needs a bit more thinking.